dart/lib/src/scope.dart

```diff
@@ -1,11 +1,8 @@
+import 'dart:async';
 import 'dart:collection';
 
-import 'sentry_attachment/sentry_attachment.dart';
-import 'event_processor.dart';
-import 'protocol.dart';
-import 'sentry_options.dart';
+import '../sentry.dart';
```

Comment:
Not a big fun of important the whole sentry file, we already have other high level files, eg, protocol, tracing, etc, maybe we import the missing one needed or add to one of those?